sections/semantics-scriptings.include

```diff
@@ -2094,71 +2124,165 @@
 
 <h5 id="sec-autonomous-custom-elements">Autonomous custom elements</h4>
 
-    An <dfn export>autonomous custom element</dfn> is a <a>custom element</a>
-    completely defined by the author. It extends {{HTMLElement}},
-    and its <a>local name</a> is equal to its <a for="customElements">name</a>.
+  An <dfn export>autonomous custom element</dfn> is a <a>custom element</a> completely defined by
+  the author. It extends {{HTMLElement}}, and its <a>local name</a> is equal to its
+  <a for="customElements">name</a>.
 
   <a>Autonomous custom elements</a> have the following element definition:
 
-<dl class="element">
-  <dt>
-    <a>Categories</a>:
-  </dt>
-
-  <dd><a>Flow content</a>.</dd>
-  <dd><a>Phrasing content</a>.</dd>
-  <dd><a>Palpable content</a>.</dd>
-
-  <dt>
-    <a>Contexts in which this element can be used</a>:
-  </dt>
-
-  <dd>
-    Where <a>phrasing content</a> is expected.
-  </dd>
-
-  <dt>
-    <a>Content model</a>:
-  </dt>
-
-  <dd>
-    <a>Transparent</a>.
-  </dd>
-
-  <dt>
-    <a>Content attributes</a>:
-  </dt>
-
-  <dd>
-    <a>Global attributes</a>, except the <{global/is}> attribute
-  </dd>
-
-  <dt>
-    <a>Allowed ARIA role attribute values</a>:
-  </dt>
-  <dd>
-    <a href="#allowed-aria-roles-states-and-properties">Any role value</a>
-  </dd>
-
-  <dt>
-    <a>Allowed ARIA state and property attributes</a>:
-  </dt>
-  <dd><a>Global aria-* attributes</a></dd>
-  <dd>
-    Any <code>aria-*</code> attributes
-    <a href="#allowed-aria-roles-states-and-properties">applicable to the allowed roles</a>.
-  </dd>
-
-  <dt>
-    <a>DOM interface</a>:
-  </dt>
-
-  <dd>Supplied by the element's author (inherits from {{HTMLElement}})</dd>
-</dl>
+  <dl class="element">
+    <dt>
+      <a>Categories</a>:
+    </dt>
+    <dd><a>Flow content</a>.</dd>
+    <dd><a>Phrasing content</a>.</dd>
+    <dd><a>Palpable content</a>.</dd>
+
+    <dt>
+      <a>Contexts in which this element can be used</a>:
+    </dt>
+    <dd>
+      Where <a>phrasing content</a> is expected.
+    </dd>
+
+    <dt>
+      <a>Content model</a>:
+    </dt>
+    <dd>
+      <a>Transparent</a>.
+    </dd>
+
+    <dt>
+      <a>Content attributes</a>:
+    </dt>
+    <dd>
+      <a>Global attributes</a>, except the <{global/is}> attribute
```

Comment:
The following should be added.
> Any other attribute that has no namespace, as determined by the element's author, so long as the attribute name is XML-compatible and contains no ASCII upper alphas. 


Ref: https://html.spec.whatwg.org/multipage/custom-elements.html#custom-elements-core-concepts:autonomous-custom-element

Reply:
thanks @mtrootyy